ci-operator/step-registry/cucushift/installer/check/vsphere/cpms/cucushift-installer-check-vsphere-cpms-commands.sh

```diff
@@ -75,15 +56,13 @@ if (( ${ocp_minor_version} < 15 )); then
 	echo "CPMS failureDomain check is only available on 4.15+ cluster, skip the check!"
         exit 0
 fi
-if (( ${ocp_minor_version} > 15 )) || [[ "${FEATURE_SET}" == "TechPreviewNoUpgrade" ]]; then
-      echo "cpms spec:"
-      oc get controlplanemachineset -n openshift-machine-api -ojson | jq -r '.items[].spec.template.machines_v1beta1_machine_openshift_io'
-      checkTemplate
-      INSTALL_CONFIG="${SHARED_DIR}/install-config.yaml"
-      readarray -t zones_setting_from_config < <(yq-go r ${INSTALL_CONFIG} 'controlPlane.platform.vsphere.zones[*]')
```

Comment:
This line seems still required for failureDomain name check.

Reply:
yes, It was originally written for multiple zones.  i will update.